core/tests/ts-integration/scripts/build.ts

```diff
@@ -0,0 +1,29 @@
+import path from 'path';
+import { exec } from 'child_process';
+import { needsRecompilation, setCompilationTime, isFolderEmpty } from './utils';
+
+const CONTRACTS_DIR = 'contracts';
+const OUTPUT_DIR = 'artifacts-zk';
+const TIMESTAMP_FILE = 'last_compilation.timestamp'; // File to store the last compilation time
```

Comment:
I think is used in multiple places, maybe you can have them in utils and import them?

Reply:
Yes, very good idea